site-styles/mixins.less

```
@@ -28,7 +28,7 @@
 // By default, xs and under use only one column.
 // "under" xs cannot be forced to anything else than 1.
 // sm and md can be configured if needed, but will default to @amount.
-#columns(@amount, @xs: 1, @sm: @amount, @md: @amount, @spacing: @gutter) {
+#columns(@amount, @xxs: 1, @xs: 1, @sm: @amount, @md: @amount, @spacing: @gutter) {
```

```suggestion
#columns(@amount, @xxs: 1, @xs: @xxs, @sm: @amount, @md: @amount, @spacing: @gutter) {
```

This way setting to e.g. `#columns(3, @xxs: 2)` sets it to 2 for `xs` too, as expected, rather than `[2, 1, 3, 3, 3]`.